From c0775294f2e6f0edd9ea5300a483b843ea194d03 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Mon, 27 Oct 2025 13:19:59 +0100
Subject: [PATCH] connections: Do not destroy Adw.Dialog

Adw.Dialog aren't windows (and don't necessarily use one), so they
don't have the destroy() method, they are closed with close() instead.

Fixes: 32d149207 ("connections: Use adaptive dialog for properties")
---
 src/connections.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/connections.js b/src/connections.js
index d2deb071..78c6983a 100644
--- a/src/connections.js
+++ b/src/connections.js
@@ -526,7 +526,7 @@ class ConnectionProperties extends Adw.Dialog {
 
         this._applyButton.connect('clicked', () => {
             this._details.save();
-            this.destroy();
+            this.close();
         });
 
         let id = account.connect('notify::connection-status',
-- 
GitLab

